back/src/domain/convention/useCases/notifications/NotifyConventionReminder.unit.test.ts

```diff
@@ -812,6 +812,7 @@ describe("NotifyThatConventionStillNeedToBeSigned use case", () => {
           ),
         );
         expectSavedEmailsNotificationsToEqual([]);
+        expectSentSmsToEqual([]);
```

Comment:
Il doit te manquer un rebase la. Tu peux utiliser la nouvelle fonction : expectSavedNotifications qlq chose...

Reply:
Un truc du genre
```suggestion
        expectSavedEmailsNotificationsToEqual({
          emails: [],
          sms: []
        })
```

ou même `expectSavedEmailsNotificationsToEqual({})` mais c'est moins explicite